homeassistant/components/light/zwave.py

```diff
@@ -20,6 +32,15 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     node = zwave.NETWORK.nodes[discovery_info[zwave.ATTR_NODE_ID]]
     value = node.values[discovery_info[zwave.ATTR_VALUE_ID]]
+    # Make sure that we have values for the key before converting to int
+    if (value.node.manufacturer_id.strip() and
```

Comment:
Uhm.... Sure. Unsure how to go about that. Because this will eventually be present for virtually all components zwave is loaded on. When things evolve it will get harder to send devices to the correct component. So some routine to recognize the correct is needed, This needs some investegation to get correct.